setup.py

```diff
@@ -1,15 +1,32 @@
 from setuptools import setup, find_packages
+from setuptools.command.install import install
+
+import os
 import setuptools
 
-# should match codalab/common.py#CODALAB_VERSION
+
+# Should match codalab/common.py#CODALAB_VERSION
 CODALAB_VERSION = "0.5.7"
 
-if int(setuptools.__version__.split('.')[0]) < 25:
-    print(
-        "WARNING: Please upgrade setuptools to a newer version, otherwise installation may break. "
-        "Recommended command: `pip3 install -U setuptools`"
+
+class Install(install):
+    _WARNING_TEMPLATE = (
+        '\n\n\033[1m\033[93mWarning! CodaLab was installed at {}, which is not\n'
+        'one of the following paths in PATH:\n\n{}\n\nConsider adding {} to be in the path in order\n'
```

Comment:
Rewrite: Warning! CodaLab was installed at {}, which is not one of the following paths in $PATH.  Consider adding {} to $PATH to use the CodaLab CLI.  You can do this by appending the following line to your `.bashrc`: {}

Reply:
@percyliang I'm not sure if can say the last sentence. I think the process is different for different OS like Windows. Should I add an entry to the FAQ of our doc? I think I can be more explicit there and give examples on how to fix the path issue in unix systems and Windows.

Reply:
If you're slick, maybe you can detect the OS can give the appropriate message.